clients/admin-ui/src/features/privacy-experience/preview/helpers.ts

```diff
@@ -71,7 +71,7 @@ export const buildBaseConfig = (
       is_default: true,
       dismissable: experienceConfig.dismissable,
       allow_language_selection: true,
-      auto_detect_language: true,
+      auto_detect_language: false,
```

Comment:
does this need to be `true` at all for the non-TCF preview?

Reply:
No, we override the language that should be shown in the preview with the `fides_locale` option (in the base preview branch, not here) so this setting isn't relevant-- will revert this change.